utils/metered_cache.go

```diff
@@ -54,6 +57,7 @@ func NewMeteredCache(size int, journal string, namespace string, updateFrequency
 		mc.gets = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/gets", namespace), nil)
 		mc.sets = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/sets", namespace), nil)
 		mc.misses = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/misses", namespace), nil)
```

Comment:
aren't some of these counts?

Reply:
gets, sets, misses and collisions are monotonically increasing unless `Reset` is called (which is not used in the codebase).
should we switch them to meters?

Reply:
I think that makes a lot more sense IMO. We won't call reset?

You may want to add a comment about that.

Reply:
nvm